keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -438,6 +446,8 @@ Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
 checking if there are objects with field X set) may be a last resort. Avoid
 logs or events for this purpose.
 -->
+By checking the StatefulSets's `.status.AvailableReplicas` field. If that field is populated
```

Comment:
isn't this set in cases when minreadyseconds==0?

they could count how many statefulsets are doing that directly, right?